R/const.R

```diff
@@ -0,0 +1,6 @@
+#' @title TAP_URL
+#' @description Link for NASA's exoplanets TAP service.
+#' @format A string.
+#' @source https://exoplanetarchive.ipac.caltech.edu/docs/TAP/usingTAP.html#bp
+#' @keywords internal
```

Comment:
suggestion: There is no real need to generate documentation for such values, this is developer-only. You can skip `.Rd` file generation by using `@noRd`.

Also, as per tidyverse guide, it is recommended that internal functions (and thus values) [have no Rd files generated](https://style.tidyverse.org/documentation.html#internal-functions).
```suggestion
#' @keywords internal
#' @noRd
```

Remember to re-run `roxygen2::roxygenise()` after that change.